src/main/java/com/example/payroll/Services/EmployeesService.java

```diff
@@ -0,0 +1,40 @@
+package com.example.payroll.Services;
+
+import com.example.payroll.Models.Employees;
+import com.example.payroll.Models.EmployeesDTO;
+import com.example.payroll.Models.EmployeesMapper;
+import com.example.payroll.Repositories.EmployeesRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class EmployeesService {
+    private final EmployeesRepository employeesRepository;
+    private final EmployeesMapper employeesMapper;
+
+    public EmployeesDTO createEmployee(EmployeesDTO employeesDTO){
+        Employees employees = employeesMapper.dtoToEntity(employeesDTO);
+        employeesRepository.save(employees);
+        return employeesMapper.entityToDto(employees);
```

Comment:
Let's extract these three lines to another method since they are completely the same

Reply:
done